<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceTabWidget</class>
 <widget class="QTabWidget" name="WorkspaceTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabImage">
   <attribute name="title">
    <string>Image</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>376</width>
         <height>247</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabTable">
   <attribute name="title">
    <string>Table</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidgetTable">
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <row/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabRawData">
   <attribute name="title">
    <string>Raw Data Preview</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEditRawData">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabHeader">
   <attribute name="title">
    <string>Header</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEditHeader">
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
